nntrainer/tensor/tensor_v2.h

```diff
@@ -370,7 +370,20 @@ class TensorV2 {
                 float value);
 
   /**
-   * @brief     Fill the Tensor elements with zero
+   * @brief     add the element value to the location
+   * @param[in] b batch location
+   * @param[in] c channel location
+   * @param[in] h height location
+   * @param[in] w width location
+   * @param[in] value value to be stored
+   * @param[in] beta scalar to multiply output with and add
```

Comment:
Just a suggestion:
How about describing params like `beta` something like:
`x = x * beta + value` for better intuition? Would it make doc more nasty?